TestMain.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <assert.h>
+#include <string>
+#include "Main.h"
```

Comment:
In Ubuntu, filenames are case-sensitive. The file is `main.h`, hence this is giving an error in the GitHub action.